javaparser-core/src/main/java/com/github/javaparser/ast/body/Modifier.java

```diff
@@ -0,0 +1,44 @@
+package com.github.javaparser.ast.body;
+
+import java.util.EnumSet;
+
+import com.github.javaparser.ast.AccessSpecifier;
+
+public enum Modifier {
+	PUBLIC("public"),
+	PRIVATE("private"),
+	PROTECTED("protected"),
+	STATIC("static"),
+	FINAL("final"),
+	SYNCHRONIZED("synchronized"),
+	VOLATILE("volatile"),
+	TRANSIENT("transient"),
+	NATIVE("native"),
+	ABSTRACT("abstract"),
+	STRICTFP("strictfp");
+

```

Comment:
Maybe go with the JLS suggested order, http://checkstyle.sourceforge.net/config_modifier.html

Doesn't technically matter, be reinforces the principal.

```
public
protected
private
abstract
static
final
transient
volatile
synchronized
native
strictfp
```